### DISCUSSION TITLE
how to enable blinking cursor in normal and insert mode

### DISCUSSION THREAD

**Discussion by shivamchandra75** [neovide/neovide#1825] [Q&A]
one of the benefits of Neovide is cursor animations so, It must have a simple blinking animation along with other cursor animations...
How can we enable blinking cursor in Neovide

**Comment by fredizzimo** [comment]
Sorry, for the very late reply, but the cursor blink is configured through standard Neovim options

https://neovim.io/doc/user/options.html#'guicursor'
